### DISCUSSION TITLE
Copilot Error in VS Code for paid Org Account: Invalid copilot token: missing token: 403

### DISCUSSION THREAD

**Discussion by rishi-nuvepro** [community/community#58599] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hello

I am facing a strange issue with two GitHub accounts enabled Copilot access when it is used in VS Code. I am getting the error highlighted below, however the same account is working in Visual Studio. I have tried multiple options - sign out, sign in, authorizing the VS Code, uninstalling and installing the Copilot extension, then I have completely removed the Copilot extension and installed the Copilot Nightly version. But nothing worked and I am unable to use both the Copilot accounts, which is giving me this error.

**[INFO] [auth] [2023-06-14T10:59:35.812Z] Invalid copilot token: missing token: 403
[ERROR] [default] [2023-06-14T10:59:35.814Z] GitHub Copilot could not connect to server. Extension activation failed: "Contact Support. You are currently logged in as copilot9-nuvepro."**

![image](https://github.com/community/community/assets/116337316/9c706fea-62a2-40a2-8bd3-277f2150101e)

I even tried VS Code in a different system to see it helps anyway. There as well I tried both the GitHub accounts by signing out from the earlier system. Looks to be a problem with these two specific GitHub user accounts that is enabled for Copilot access, and that too not working in VS Code but working in Visual Studio. When I tried Copilot in Visual Studio, the same GitHub login and the same system, worked perfect!! - _see the screenshot given below_. We have many GitHub Copilot accounts, which is working without any issues.

![image](https://github.com/community/community/assets/116337316/ee497a00-35ce-4ce4-a21f-c8829349a6a9)

Any help would be appreciated!!

**Comment by davecheney** [comment]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by rishi-nuvepro** [reply]
Thank you. But I have tried all these options as I have detailed in my description above. If I sign in with a working GitHub account in the same VS Code, it is working. As soon as I login with this specific account, I start seeing the error - **Invalid copilot token: missing token: 403**. 

Two of our accounts are not working failing with the same reason and only in VS Code. It works perfect in Visual Studio as given in the screenshot above. Even if I try those accounts in VS Code in a different machine, I am getting the same error. All other accounts enabled for Copilot is working without any issues.

**Reply by davecheney** [reply]
@rishi-nuvepro `copilot9-nuvepro` has been flagged for account review. Please contact support, https://support.github.com/contact.

**Reply by rishi-nuvepro** [reply]
Could you please let me know what is meant by _flagged for account review_? I have wrote to Support about this issue facing with copilot9 & copilot11 accounts. I haven't received any update despite waiting for last 7 days. The ticket is still in open status. Appreciate your support!!

**Reply by davecheney** [reply]
I'm sorry I cannot assist you further and also apologise for the delay.

**Reply by rishi-nuvepro** [reply]
Could you please help me understand what do you meant by flagged for account review?

![image](https://github.com/community/community/assets/116337316/bd118d8d-2233-41a0-ad9d-63d3cd2aefdf)

**Reply by davecheney** [reply]
I'm sorry I cannot discuss that on this forum. You must contact support.

**Reply by rishi-nuvepro** [reply]
Thank you. I will check with support. I hope they would reply this time.

**Comment by rishi-nuvepro** [comment]
Account was reset by the GitHub team and now it is working perfect!
